src/OneExplorer/ConfigObject.ts

```diff
@@ -25,21 +25,26 @@ import { Logger } from "../Utils/Logger";
 
 import { Artifact } from "./ArtifactLocator";
 import { ConfigSetting } from "./ConfigSetting";
-import { OneConfigSetting } from "./ConfigSettings/OneConfigSetting";
-import { EdgeTpuConfigSetting } from "./ConfigSettings/EdgeTpuConfigSetting";
+import { OneCfg, OneConfigSetting } from "./ConfigSettings/OneConfigSetting";
+import {
+  EdgeTpuCfg,
+  EdgeTpuConfigSetting,
+} from "./ConfigSettings/EdgeTpuConfigSetting";
 import { BackendContext } from "../Backend/API";
 
-type Cfg = {
-  "one-import-tflite": CfgOneImportTflite;
-  "one-import-onnx": CfgOneImportOnnx;
-  "one-import-tf": CfgOneImportTf;
-};
+type Cfg = OneCfg & EdgeTpuCfg;
 type CfgKeys = keyof Cfg;
 
 // TODO Update
-type CfgOneImportTflite = any;
-type CfgOneImportOnnx = any;
-type CfgOneImportTf = any;
+// type CfgOneImportTflite = any;
+// type CfgOneImportOnnx = any;
+// type CfgOneImportTf = any;
+
+type CfgType = {
+  default: any;
```

Comment:
I think this type's naming is "one" better than  "default" because "default" is ambiguous.